### PR TITLE
Added docs for JSON helper

### PR DESCRIPTION
**Pull request by dfridrich**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 3.1 |
| Fixed tickets | #6425 |

### CODE REVIEW

File: book/controller.rst [hunk 1]
```diff
@@ -778,6 +778,20 @@ There are also special classes to make certain kinds of responses easier:
   :class:`Symfony\\Component\\HttpFoundation\\StreamedResponse`.
   See :ref:`streaming-response`.
 
+JSON helper
+~~~~~~~~~~~
+
+You can simplify sending JSON response with
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::json`
+helper::
```
Comment (javiereguiluz) [inline book/controller.rst]: I miss here some information about the argument/arguments expected by this method.

Comment (dfridrich) [inline book/controller.rst]: I will update PR in a while.

**Comment by javiereguiluz** [comment]
In #6502 I'm proposing a different way to add this doc. @dfridrich your explanation is correct, but I think it comes too early in the chapter. I propose to move it to the section explaining the base controller.

**Comment by dfridrich** [comment]
@javiereguiluz I agree with you.

File: book/controller.rst (hunk 1 above)
Comment (WouterJ) [inline book/controller.rst]: The greatest feature of this shortcut is the usage of the Symfony serializer if available. We surely have to document that imo

**Comment by WouterJ** [comment]
What about moving this note after the http://symfony.com/doc/current/book/controller#forwarding-to-another-controller section? It seems a better place to me than the one proposed in #6502

**Comment by WouterJ** [comment]
I'm closing this one because https://github.com/symfony/symfony-docs/pull/6502 was merged. Still, I would thank you for helping thinking about the best location to document this feature. I hope to see you back with other PRs!

**Pull request closed without merging**